master/buildbot/interfaces.py

```diff
@@ -60,11 +60,20 @@ class LatentWorkerFailedToSubstantiate(Exception):
     pass
 
 
+class LatentWorkerCannotSubstantiate(Exception):
+    pass
+
+
 deprecatedWorkerModuleAttribute(
     locals(), LatentWorkerFailedToSubstantiate,
     compat_name="LatentBuildSlaveFailedToSubstantiate")
 
 
+deprecatedWorkerModuleAttribute(
```

Comment:
no need for that. we never released a LatentBuildSlaveCannotSubstantiate symbol

Reply:
Will Remove